lang/en/accredible.php

```diff
@@ -78,3 +78,8 @@
 $string['overview'] = 'Overview';
 $string['activitygroupdescription'] = 'Credentials groups need to have been created in the <a href="{$a}" target="_blank">Accredible Dashboard</a> before credentials can be issued. If none appear, check your API Key to make sure integration is set up correctly.';
 $string['accrediblegroup'] = 'Accredible Group';
+
+$string['privacy:metadata:accredible'] = 'In order to integrate with Accredible, user data needs to be exchanged with that service.';
+$string['privacy:metadata:accredible:email'] = 'Your email address is sent to Accredible to issue a credential.';
+$string['privacy:metadata:accredible:fullname'] = 'Your full name is sent to Accredible to issue a credential.';
+$string['privacy:metadata:accredible:quizgrade'] = 'Your quiz grade is sent to Accredible to issue a credential.';
```

Comment:
Can we reword this to:
Your quiz grade **may be** sent to Accredible to issue a credential.

Reply:
That's a good point as we're also planning to add a new setting option to disable evidence item creation. I updated the wording: https://github.com/accredible/moodle-mod_accredible/pull/58/commits/b673d36a49b5be0b395ed37f9e9fca83633dd648

![Screen Shot 2022-04-20 at 18 46 35](https://user-images.githubusercontent.com/20009662/164201189-3a41de0a-a187-48e2-b9f7-4094f602e2e9.png)